README.rst

```diff
@@ -522,6 +522,14 @@ Choose from the list of available fixers:
                         Align equals symbols in
                         consecutive lines.
 
+* **array_element_space** [contrib]
+                        In array declaration, there
+                        MUST NOT be a space before
+                        each comma, there MUST be one
+                        space after each comma and
+                        there should be space around
```

Comment:
I don't like that the fixer add spaces around `=>` or... shorten it.
It is not what was described here.
We already play a lot with `=>`, so the fixers are touching same thing in different way.
Please split this fixer into to - one that handle `,`, one that handle `=>`